modules/dnn/src/cuda/mvn.cu

```diff
@@ -77,6 +77,27 @@ namespace raw {
             output[idx] = (static_cast<float>(input[idx]) - means[outer_idx]) * s + b;
         }
     }
+
+    template <class T>
+    __global__ void normalize_mean_variance_layernorm(Span<T> output, View<T> input, View<T> scale, View<float> means, View<float> stdev, size_type inner_size) {
+        for (auto idx : grid_stride_range(output.size())) {
+            const index_type outer_idx = idx / inner_size;
+            const index_type inner_idx = idx % inner_size;
+            auto s = static_cast<float>(scale[inner_idx]) * stdev[outer_idx];
```

Comment:
`inv_stddev`?

Reply:
it is already inverted in the above kernel `compute_normalization_scale`.

Reply:
I mean just a name for a variable. It's expected to multiply on an inverse deviation.

Reply:
Ok.